### ISSUE TITLE
Dynamic Everywhere not working in Link (URL)

### ISSUE THREAD

**Issue by bislamovic** [dudaster/ele-custom-skin#247]
Hi,

 I have strange problem. After I migrated my site from Dev to Production I get this problem.

I have posts (slider) with customs skin created with your plugin. Each element inside is filled with #anchore links - link is created with Dynamic Everywhere  `#{{ID}}.`
![image](https://user-images.githubusercontent.com/21058182/85205811-f8558880-b31d-11ea-945f-f61fbfc77ddd.png)

I have another posts widget on the same page with custom skin which use anchore with HTML code: `<a id="{{ID}}"></a>`
![image](https://user-images.githubusercontent.com/21058182/85205797-e247c800-b31d-11ea-8e56-3cb6f3ccb8aa.png)

Idea is to have page dynamic navigation.
It worked nice until migration. Now Dynamic Everywhere is not show right only in Links. Do you have any idea?

link: https://safirhoteli.hr/dining/

**Comment by dudaster** [comment]
yeah... from the last update from elementor is not working. I'm working on a fix to use shortcodes instead of keywords for the links.
